client/views/analytics/index.html

```diff
@@ -0,0 +1,105 @@
+<div class="sd-page-grid--basic">
+    <header class="sd-top-menu">
+        <a href="" class="sd-top-menu__collapse-nav"><i class="icon-collapse icon--white"></i></a>
+        <p class="sd-top-menu__header">Publisher</p>
+    </header>
+
+    <section class="sd-content sd-page-content--slide-in">
+        <div class="sd-loader" ng-if="loading"></div>
+        <div class="sd-sidebar-menu sd-sidebar-menu--second-level sd-page-content__left-tabs" ng-include="'sidebar-content.html'"></div>
+        <div class="sd-page-content__content-block">
+            <div class="subnav">
+                <a href="#/publisher/dashboard" class="navbtn navbtn--left">
+                    <i class="icon-arrow-left"></i>
+                </a>
+                <h3 class="subnav__page-title">Content Analytics</h3>
+            </div>
+            <div class="sd-column-box--3">
+                <div class="sd-content-navigation-panel sd-content-navigation-panel--border-right overflow-auto">
+                    <div class="subnav subnav--padded subnav--lower-z-index">
+                        <h3 class="sd-content-nav-title">Website</h3>
+                    </div>
+                    <nav class="sd-content-nav">
+                        <ul>
+                            <li ng-repeat="site in webPublisherAnalytics.sites">
+                                <a href="" class="sd-content-nav__btn"
+                                    ng-click="webPublisherAnalytics.setTenant(site)"
+                                    ng-class="{'sd-content-nav__btn--active' : webPublisherAnalytics.selectedTenant.code === site.code}">
+                                    {{:: site.name }}</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+                <div class="sd-column-box__main-column sd-column-box__main-column--flex">
+                   <!-- <div class="subnav subnav--lower-z-index">
```

Comment:
do we need to merge that commented code?

Reply:
yes